core/index.html

```diff
@@ -1389,67 +1389,98 @@
       </div>
     </figure>
 
-      <!-- <section id="defaultlangcontext">
+      <section id="defaultlangcontext">
         <h3>Default Language Context</h3>
 
+        <p>
+          The JSON format does not provide a clear mechanism for identifying the
+          default language for natural-language properties such as
+          <code>name</code>, <code>content</code>, and <code>summary</code>.
+        </p>
+
+        <p>
+          When using [[JSON-LD]] mechanisms to produce or consume Activity
+          Streams 2.0 documents, the <code>@language</code> property MAY be used
+          within a <code>@context</code> to identify the default language. This
+          mechanism may not be understood by implementations that do not choose
+          to process Activity Streams 2.0 documents using JSON-LD.
+        </p>
+
+        <p>
+          Because of the importance of identifying the language of natural
+          language properties, it is RECOMMENDED that implementations use the
```

Comment:
I don't think this should be preferred. The vast majority of AS documents will be unilingual, and using a map for every single string property will be cumbersome and unnecessary. Using the global @context["@language"] is much clearer.